test/resolver.spec.js

```diff
@@ -77,19 +78,36 @@ describe('IPLD format resolver (local)', () => {
 
         expect(paths).to.eql([
           'name',
+          'nest',
```

Comment:
I'm not a 100% sure on the changes to this test case, why are these the correct ones/what was wrong about the ones before?